contracts/arbitrum/connectors/uniswap-sell-beta/main.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+import "./helpers.sol";
+
+abstract contract uniswapSellBeta is Helpers {
+    function sell(
+        address tokenIn,
+        address tokenOut,
+        uint24 fee,
+        uint256 amountIn,
+        uint256 amountOutMinimum,
+        bool zeroForOne
+    ) public returns (uint256 amountOut) {
+        approveTransfer(tokenIn, msg.sender, address(this), amountIn);
+        amountOut = getSingleInput(
```

Comment:
Is this where the swap is happening? If so then never keep the function as `get` as that is mainly referred to a function that just returns some data without updating any storage.

Reply:
You can maybe rename it to - `swapSingleInput`

Reply:
ok sure